anvio/workflows/metagenomics/Snakefile

```diff
@@ -1029,27 +1029,35 @@ rule import_percent_of_reads_mapped:
                     anvi-import-misc-data {output.total_num_reads_txt} \
                                           -p {input.profile} \
                                           -t layers \
-                                          --just-do-it
+                                          --just-do-it >> {log} 2>&1
 
                     # get the layers additional data table, which has two columns
                     # now.
                     anvi-export-misc-data -p {input.profile} \
                                           -t layers \
-                                          -o {output.layers_additional_data}
-
-                    # do an awk one-liner to add a third column, 'percent mapped'.
-                    awk '{{if(NR==1) \
-                             {{print $0 "\tpercent_mapped"}} \
-                          else \
-                              {{print $0 "\t" ($2 * 100 / $4)}}}}' \
-                        {output.layers_additional_data} > {output.layers_additional_data_updated}
+                                          -o {output.layers_additional_data} >> {log} 2>&1
+                  """)
+
+            additional_data = pd.read_csv(output.layers_additional_data, sep='\t', index_col=None)
+            if 'total_reads_mapped' not in additional_data.columns:
+                raise ConfigError('The layers additional data for %s doesn\'t contain a column
```

Comment:
ignorant question, is this really going to work without the new-line escapes (`/`)?

Reply:
I don't think it would :-) luckily if someone got there, they deserve an error (just not the error that they will get)